packages/terra-graphs-docs/package.json

```diff
@@ -35,8 +35,8 @@
     "terra-theme-context": "^1.0.0"
   },
   "devDependencies": {
-    "terra-button": "^3.55.0",
-    "@cerner/terra-dev-site": "^8.0.0"
+    "@cerner/terra-dev-site": "^8.0.0",
```

Comment:
Looks like the version did not change and is still at 8.0. Is it correct?

Reply:
yea, the line says its changed cuz it added a comma cuz it moved terra-button below for alphabetical order